### DISCUSSION TITLE
Support for Postgres JSON operators

### DISCUSSION THREAD

**Discussion by bbqsrc** [SeaQL/sea-orm#2150] [Ideas]
A number of my use cases require dealing with a bag of properties, and querying for the existence of things in said bag. Maybe even using the resulting value on occasion, haha.

Supporting the operators on [this page](https://www.postgresql.org/docs/current/functions-json.html) would be helpful to my use case, particularly `->` and `->>`.

Is there any known workaround I can use in the meantime?

**Comment by anshap1719** [comment]
@bbqsrc Can you please describe your use case? I'm pretty sure that this is already supported with https://docs.rs/sea-query/latest/sea_query/extension/postgres/trait.PgExpr.html#method.get_json_field & https://docs.rs/sea-query/latest/sea_query/extension/postgres/trait.PgExpr.html#method.cast_json_field.

It's probably just not documented with sea-orm, but I'm already using it in a project to run queries agains json fields, so I know it works.

**Reply by bbqsrc** [reply]
Ohhh. That’ll likely do it. Thanks!
